### DISCUSSION TITLE
Randomiser les alertes

### DISCUSSION THREAD

**Discussion by Dannou** [skarab42/marv#48] [Ideas]
la possibilité d'avoir plusieurs animations différentes randomiser pour le même évent (ex follow, Sub)

**Comment by Dannou** [comment]


**Reply by Dannou** [reply]
Oui fort bien. 
Il n'y a guère longtemps que je ne me suis investi de la sorte, au delà de potron-minet sur une tel idees.
hahaha je m'enjaille.

**Reply by Dannou** [reply]
ou bien ?

**Reply by Dannou** [reply]
ou bien quoi ?

**Reply by Dannou** [reply]
exactement

**Reply by ghost** [reply]
interressent

**Comment by skarab42** [comment]
Yep c'est dans la todo ;)
